### DISCUSSION TITLE
[Feature] Add to album for non-uploaded items

### DISCUSSION THREAD

**Discussion by guyfischman** [immich-app/immich#14935] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I take a photo, and want to add it to an immich album. Now I have to select it, upload, and confirm I want to upload. Then select it, and add it to an album.

I want to just select it and add it to the album, and have the upload happen as a result.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#12190
